cloud_edition/serenity_mode/docs/overview.rst

```diff
@@ -0,0 +1,19 @@
+Overview
+--------
+
+To start, you need to set up your serenity instances from the `Portal <https://help.akeneo.com/portal/articles/set-up-akeneo-serenity.html?utm_source=akeneo-docs&utm_campaign=serenity_overview>`_
+
+Akeneo Cloud Serenity provides 2 environments:
+
++--------------------------------------+--------------------------------------+
+| - The **production** environment:    | my-project.cloud.akeneo.com          |
++--------------------------------------+--------------------------------------+
+| - The **sandbox** environment:       | my-project-staging.cloud.akeneo.com  |
++--------------------------------------+--------------------------------------+
+
+.. warning::
+
+    **Warning:** Only the production environment is backuped. All data produced on the sandbox environment are not backed up.
+
+.. note::
```

Comment:
seems to obvious to be stated but why not after all ;)

Reply:
I take as model the flexibility page.